packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -29,8 +29,8 @@ declare const getStatusLambda: lambda.Function;
 
 const submitJob = new tasks.LambdaInvoke(this, 'Submit Job', {
   lambdaFunction: submitLambda,
-  // Lambda's result is in the attribute `Payload`
-  outputPath: '$.Payload',
+  // Lambda's result is in the attribute `guid`
+  outputPath: '$.guid',
```

Comment:
`$.guid` is later used here https://github.com/aws/aws-cdk/pull/24117/files#diff-c3ac3f3bd0729bf5d5d41f940534332445cf6827fec30c0d7e98693c556e6d9aR44